src/game/engine/player.py

```diff
@@ -26,6 +30,10 @@ class Player:
     def __init__(self, position, image_path: str):
         self.horizontal_vel=50
         self.position = position
+        self.image_path= pygame.transform.scale(pygame.image.load(f"{PATH_TO_DIR}{os.sep}assets{os.sep}game{os.sep}horsey.png"),(position.width,position.height))
+        self.controls = {pygame.K_RIGHT:"right",pygame.K_LEFT:"left",  pygame.K_SPACE:"jump"}
```

Comment:
we should have this be a `Dict[pygame.K_RIGHT, Callable]` rather than `str` (Callable is a function). I believe that's what it is on `Master`.